core/frontend/src/components/vehiclesetup/configuration/compass/CalibrationQualityIndicator.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div ref="bar" class="color-bar">
+    <span
+      ref="display"
+      :style="{ left: computed_left + 'px', 'background-color': computed_color }"
+      class="signal-quality"
+    >
+      {{ quality.toFixed(0) }}
+    </span>
+  </div>
+</template>
+
+<script lang="ts">
+
+export default {
+  name: 'CalibrationQualityIndicator',
+  props: {
+    quality: {
```

Comment:
Can you add a comment explaining what range quality is working here ?

Reply:
done